internal/admission/server.go

```diff
@@ -287,6 +292,17 @@ func (a RequestHandler) handleValidation(ctx context.Context, request admission.
 		if err != nil {
 			return nil, err
 		}
+	case httprouteGVResource:
+		httproute := gatewayv1alpha2.HTTPRoute{}
+		deserializer := codecs.UniversalDeserializer()
+		_, _, err = deserializer.Decode(request.Object.Raw, nil, &httproute)
+		if err != nil {
+			return nil, err
+		}
+		ok, message, err = a.Validator.ValidateHTTPRoute(context.Background(), httproute)
```

Comment:
Can we make a drive-by fix to use the right `context.Context` in this function?

Reply:
:+1: fixed in 40c231c2d145f61773b6522d1ba5fc39a9e8a9e0